lib/reference/atKeywords.js

```diff
@@ -54,4 +58,5 @@ const atKeywords = uniteSets(pageMarginAtKeywords, [
 
 module.exports = {
 	atKeywords,
+	nestingSupportedAtKeywords,
 };
```

Comment:
offtopic

maybe we could replace `uniteSets` by https://www.npmjs.com/package/set.prototype.union?

Reply:
Interesting. But I'd like to minimize dependencies as possible.